static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -63,7 +69,7 @@ const PublishPreview = () => {
       const {
         revisionId: savedRevisionId,
```

Comment:
Maybe prefix this with fetched instead of saved?
```suggestion
        revisionId: savedRevisionId,
```